docs/guides/vtex-io/Storefront-Guides/getting-started-with-storefront-solutions.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Getting started with storefront solutions"
+slug: "getting-started-with-storefront-solutions"
+hidden: false
+excerpt: ""
+createdAt: "2024-05-20t12:59:27.687z"
+updatedAt: "2024-05-20t12:59:27.687z"
+---
+
+The VTEX platform offers three options for storefront development, each with distinct characteristics to meet the needs of our clients:
+
+- **FastStore**: Innovative technology that prioritizes performance and stability, offering development teams an easily maintainable solution for editing store pages.
+- **Store Framework**: Emphasizes a low-code, composable commerce model, allowing integration of VTEX IO apps for dynamic storefronts.
+- **Legacy CMS Portal**: VTEX’s inaugural storefront technology, that relies on HTML, CSS, and JavaScript, with code management exclusively through the VTEX Admin.
+
+The following table outlines the main differences between VTEX’s Legacy CMS Portal, Store Framework, and FastStore.
+
+|     |     |     |     |
+| --- | --- | --- | --- |
+|     | <div align="center">**FastStore**</div> |<div align="center">**Store Framework**</div> | <div align="center">**Legacy CMS Portal**</div> |
+| **Storefront development** | FastStore is built with a \*\*performance\*\* focus based on the Jamstack (JavaScript, API, and Markup) architecture.<br><br>The development is carried out through the integration of \`npm\` packages provided by VTEX and customizations via the extensibility of VTEX API core services. | Store Framework is based on \*\*composability\*\*, as its development is based on VTEX IO apps, allowing the combination of different content, services, and data. | The development is carried out through the Admin, using the [Legacy Content Management System (CMS)](https://help.vtex.com/en/tracks/cms--2YcpgIljVaLVQYMzxQbc3z/6OCY6S9tqBXPD5mgpbBInC). |
+| **Tech stack** | * React<br>* TypeScript<br>* Node.js<br>* Next.js<br>* GraphQL<br>* VTEX WebOps | * React<br>* TypeScript<br>* Node.js<br>* .NET<br>* GraphQL<br>* VTEX IO | * CSS<br>* HTML<br>* JavaScript |
+| **Starter (initial template)** | After completing the [FastStore Starter](https://starter.vtex.app/), you can access the [FastStore Onboarding](https://www.faststore.dev/docs/getting-started-overview). This allows you to kick off the project with a base template focused on performance. | The [Store Theme](https://github.com/vtex-apps/store-theme) is the initial template you can use to launch the storefront project or start from scratch. | The [Layout](https://help.vtex.com/es/tracks/cms--2YcpgIljVaLVQYMzxQbc3z/1oN446gRGcR2s70RvBCAmj) section allows you to create your storefront with HTML and CSS.<br><br>Inside it, you can find the standard [templates](https://help.vtex.com/en/tutorial/what-are-templates--4l7BQBYO9ycumsqua2CU88), which are files that contain the code of your website pages. |
```

Comment:
Na FastStore, para pessoa ter o Starter da sua loja ela primeiro passa pelo processo do Onboarding no WebOps e no final ela tem o starter. Talvez podemos ter uma frase como: `The [Starter Store](https://starter.vtex.app/) becomes your initial store template, once you complete the [FastStore WebOps onboarding](https://developers.vtex.com/docs/guides/faststore/getting-started-overview).`